osquery/tables/system/ssh_keys.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <string>
+#include <vector>
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/filesystem.h>
+
+#include "osquery/tables/system/system_utils.h"
+
+namespace osquery {
+namespace tables {
+
+const std::string kSSHUserKeysDir = ".ssh/";
+
+void genSSHkeyForHosts(const std::string& uid,
+                        const std::string& directory,
+                        QueryData& results) {
+  //get list of files in directory
```

Comment:
nitpicking: your comment style is inconsistent. Either capitalize the start of each, or don't, but please don't mix. Also, it's best to begin each with a space after the double-slash: `// `.